__tests__/config.ts

```diff
@@ -0,0 +1,31 @@
+/*
+* Copyright 2022 Mia srl
```

Comment:
can we please fix the indentation of these comments by padding with one space these asterisks